config/config.go

```diff
@@ -99,6 +100,17 @@ func DefaultConfig() string {
 	return proto.MarshalTextString(&configpb.ProberConfig{})
 }
 
+type OptionalString struct {
```

Comment:
As I mentioned in the comment on the PR as well, it's not clear to me why are you using OptionalString?